sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/Helpers/KnownMethods.cs

```diff
@@ -39,6 +39,7 @@ public static bool IsMainMethod(this IMethodSymbol methodSymbol)
                     (methodSymbol.Parameters.Length == 1 && methodSymbol.Parameters[0].IsType(KnownType.System_String_Array)));
         }
 
+        #region --- Object Methods ---

```

Comment:
Region here suggests we should split the class to something like KnownObjectMethods, KnownEnumMethods, etc. let's leave it for now though.